src/organisms/hintsAndHovers/hintsAndHovers.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { HintsAndHovers } from './hintsAndHovers';
+import { Styled } from '../../theme';
+
+export default {
+  title: 'Organisms/HintsAndHovers',
+  component: HintsAndHovers,
+  argTypes: {
+  },
+} as ComponentMeta<typeof HintsAndHovers>;
+
+const Wrapper = Styled.div`
+  margin-left: 100px;
```

Comment:
With margin-left of 100px the `left` popover actually displays on the top. If you increase the margin then the left works fine, so to demonstrate all options maybe it is better to increase the margin here